### DISCUSSION TITLE
Images for Pedal Keyboard selection 2

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#41] [General]
I think the images for the natural keys of the pedalboard (image selection 2) need some improvement. They do not show well and it makes it look like the pedal keyboard is made up of just the sharp keys.

**Comment by larspalo** [comment]
The pedal images are part of GrandOrgue, not GOODF. If you don't like them - don't use them, or suggest improvements in GrandOrgue - but be ready to make/supply those improvements then!
